### PR TITLE
Show user data for newly registered users

### LINKED ISSUE #203: Show newly registered user's data on admin page
The 'show' button currently not working...

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #203

### CODE REVIEW

**Review by kdmnk — CHANGES_REQUESTED** [review]
If we want this in a separate page, it would be practical to place the approve or deny buttons for this page too. 
And when you click one of them, the next user could appear (useful for mass handling) - but we can open an issue for this.

File: resources/views/admin/user.blade.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                            </tr>
+                            <tr>
+                                <th scope="row">@lang('info.year_of_graduation')</th>
+                                <td>{{ $user->educationalInformation->year_of_graduation }}</td>
+                            </tr>
+                            <tr>
+                                <th scope="row">@lang('info.year_of_acceptance')</th>
+                                <td>{{ $user->educationalInformation->year_of_acceptance }}</td>
+                            </tr>
+                        @endif
```
Comment (kdmnk) [inline resources/views/admin/user.blade.php:101]: The same can be copied to the user page. Please open an issue for this.

**Comment by luksan47** [comment]
> If we want this in a separate page, it would be practical to place the approve or deny buttons for this page too.
> And when you click one of them, the next user could appear (useful for mass handling) - but we can open an issue for this.

Added the buttons to the page. Now they redirect back to the list, will open an issue for your suggestion.

Also, there are some code duplicates I don't like. Once we won't be in a hurry, we should remove these.

**Review by kdmnk — APPROVED** [review]
Ok for now.
Please open further issues, including https://github.com/luksan47/mars/pull/282/files#r478695138 and mass handling registrations.

**Pull request merged**